src/templates/header-basic-mega.html

```diff
@@ -1,173 +1,11 @@
 <header class="usa-header usa-header-basic usa-header-basic-megamenu" role="banner">
   <div class="usa-nav-container">
-    <div class="usa-navbar">
-      <button class="usa-menu-btn">Menu</button>
-      <div class="usa-logo" id="logo">
-        <em class="usa-logo-text">
-          <a href="#" title="Home" aria-label="Home">Department of <br>Web Standards</a>
-        </em>
-      </div>
-    </div>
+    {% include 'masthead.html' %}
+
     <nav role="navigation" class="usa-nav">
```

Comment:
I would kinda expect the nav component to include a `nav` element, but it also sounds like it would be hard to split them up with it? What were your thoughts on this?